Jenkinsfile

```diff
@@ -141,23 +141,6 @@ try {
           tool: esLint(id: 'eslint', name: 'eslint', pattern: 'codestyle-fe.xml'),
           trendChartType: 'NONE'
         )
-
```

Comment:
Kindly restore this mandatory part. As you can see, the pipeline failures are not related to this step